kevinb5617/ResuMake
# Added package-lock to vcs and updated gitignore

# Description

The `.gitignore` file did not have the correct files in it, according to React "standard"

# Related Issue(s) and Pull Request(s)

#9

# Solution

Added massive `node_modules/` directory to `.gitignore` and removed `package-lock.json`

# Steps to Test

1. `git fetch upstream`
2. `git checkout upstream/feature/fix-gitignore`
3. Add a test file to `app/node_modules`
4. `git status`
5. You should see that change to `app/node_modules` is not tracked by git
6. Remove the test file

# Known Issues

None

# Checklist
